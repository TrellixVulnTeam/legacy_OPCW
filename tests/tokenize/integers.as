3242324
3462778959793846834
4352345
4352.slap()
324673289732978
0b01101011010
0B10101
0x0FABCDE
0xdeadbeef.bits()
0X320942
0O1123214
0o755
0o755.some_method()
