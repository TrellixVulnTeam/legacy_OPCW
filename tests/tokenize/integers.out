<stdin>:1:1-8: decimal integer '3242324'
<stdin>:2:1-20: decimal integer '3462778959793846834'
<stdin>:3:1-8: decimal integer '4352345'
<stdin>:4:1-5: decimal integer '4352'
<stdin>:4:5-6: punctuator: '.'
<stdin>:4:6-10: identifier 'slap'
<stdin>:4:10-11: punctuator: '('
<stdin>:4:11-12: punctuator: ')'
<stdin>:5:1-16: decimal integer '324673289732978'
<stdin>:6:1-14: binary integer '01101011010'
<stdin>:7:1-8: binary integer '10101'
<stdin>:8:1-10: hexadecimal integer '0FABCDE'
<stdin>:9:1-11: hexadecimal integer 'deadbeef'
<stdin>:9:11-12: punctuator: '.'
<stdin>:9:12-16: identifier 'bits'
<stdin>:9:16-17: punctuator: '('
<stdin>:9:17-18: punctuator: ')'
<stdin>:10:1-9: hexadecimal integer '320942'
<stdin>:11:1-10: octal integer '1123214'
<stdin>:12:1-6: octal integer '755'
<stdin>:13:1-6: octal integer '755'
<stdin>:13:6-7: punctuator: '.'
<stdin>:13:7-18: identifier 'some_method'
<stdin>:13:18-19: punctuator: '('
<stdin>:13:19-20: punctuator: ')'
<stdin>:14:1-2: end
